src/pages/CommitDetailPage/CommitBundleAnalysis/BundleMessage/BundleMessage.tsx

```diff
@@ -47,9 +47,9 @@ const BundleMessage: React.FC = () => {
     comparison?.__typename === 'BundleAnalysisComparison' &&
     isNumber(comparison?.bundleChange?.size?.uncompress)
   ) {
-    const sizeDelta = comparison.bundleChange.size.uncompress
-    const positiveSize = Math.abs(sizeDelta)
-    if (sizeDelta < 0) {
+    const uncompressDelta = comparison.bundleChange.size.uncompress
```

Comment:
any reason why we called this const uncompressDelta vs uncompress? Same with another instance below

Reply:
Just to be a bit more explicit that it's a change in size instead of just a static size

Reply:
Gotcha, thanks for the context!